llvm/test/tools/llvm-symbolizer/no-use-symbol-table.s

```diff
@@ -0,0 +1,219 @@
+# REQUIRES: x86-registered-target
+
+# RUN: llvm-mc -filetype=obj -triple=x86_64-pc-linux %s -o %t.o
+# RUN: llvm-addr2line --no-use-symbol-table -fe %t.o 0x1 | FileCheck %s
+# CHECK: foo
+# CHECK: relocatable.c
+# The above addr2line command prints "b" if --use-symbol-table
```

Comment:
New tests prefer `##` to distinguish test comments from actual test commands.

Also, the same coding guidelines about comments apply to code as well as to tests, i.e. here and below missing a full stop.

I think it would be useful to briefly explain the why behind the difference in behaviour.